lib/trailblazer/rails/railtie.rb

```diff
@@ -36,6 +36,7 @@ def self.load_for(app)
     initializer "trailblazer.application_controller" do
       reloader_class.to_prepare do
         ApplicationController.send :include, Trailblazer::Rails::Controller
+        ApplicationController.send :include, Trailblazer::Rails::Controller::Cell if defined?(::Cell)
```

Comment:
I need - I just don't have a base controller for rails with the app controller name ;(